### ISSUE TITLE
Trying to jump to a route page with scss file imported from a route page without scss, it blocked and i can not get my goal page

### ISSUE THREAD

**Issue by ghostTing** [vercel/next-plugins#531]
```typescript
// index.tsx without scss file imported
import * as React from 'react'
import Link from 'next/link'
import Layout from '../components/Layout'
import { NextPage } from 'next'

const IndexPage: NextPage = () => {
  return (
    <Layout title="Home | Next.js + TypeScript Example">
      <h1>Hello Next.js 👋</h1>
      <p>
        <Link href="/about">
          <a>About</a>
        </Link>
      </p>
    </Layout>
  )
}
```
```typescript
// about.tsx with scss imported
import * as React from 'react'
import Link from 'next/link'
import Layout from '../components/Layout'
import WithReactIntl from '../i18n/with-react-intl'
import {FormattedMessage} from 'react-intl'
import style from '../styles/about.scss'

const AboutPage: React.FunctionComponent = () => (
  <Layout title="About | Next.js + TypeScript Example">
    <h1 className={style.about}>About</h1>
    <p>This is the about page</p>
    <p>
      <Link href="/">
        <a>Go home</a>
      </Link>
    </p>
  </Layout>
)
```
```javascript
// next.config.js
const withSass = require('@zeit/next-sass')
console.log('next.config.js')
module.exports = withSass({
  cssModules: true,
  cssLoaderOptions: {
    importLoaders: 1,
    localIdentName: "[local]___[hash:base64:5]",
  }
})
```
i can get my goal page when typing url 'localhost:3000/about' and the scss style become effective. however when using <Link> to routing page about, it first sucking at 
```
 event ] build page: /about
[ wait ]  compiling ...
[ info ]  bundled successfully, waiting for typecheck results ...
```
then turn to 
```
[ event ] disposing inactive page(s): /next/dist/pages/_error, /about
[ ready ] compiled successfully
```
but my browser get nothing even any warnings or errors
ps: it happen when using @zeit/next-less instead

**Comment by wonsz1** [comment]
Well, as far as I know, you need to add empty css file to every page xD

**Comment by pixelass** [comment]
I ran into this issue and it seems like this is entirely broken.

I imported an empty css file in my `pages/_app.tsx` to fix it.

IMHO this should have priority and be fixed soon since it basically breaks every page that imports a css file without having a previous page that imports CSS.

In my case the links worked after I visited those routes before clicking the links. (After e page reload they stop working again)

Without visiting before  
![next-link](https://user-images.githubusercontent.com/1148334/70508551-f0c35480-1b2e-11ea-9304-07ba50f13cd9.gif)


With visiting before  
![next-link_2](https://user-images.githubusercontent.com/1148334/70508557-f5880880-1b2e-11ea-8406-1ff028cecb4a.gif)


It looks like there are several people reporting this in different issues with different configs, so I'm not going to open another one here.

End of the story is: IT'S BROKEN
